### DISCUSSION TITLE
User Story from OS (24-02-2020) #2

### DISCUSSION THREAD

**Discussion by GeorgiaHCA** [alan-turing-institute/AutSPACEs#367] [General]
### I would like 

the platform to last for a long time 

### Because...

it can expand and grow in a positive way 

### Acceptance criteria

There is a plan in place for after the official involvement and funding ends
There is a community around the platform
There is clear documentation and on boarding processes

---

## Notes

* Associated pull request: 
* Current status:

---

## Updates

**Comment by fjThomasStanley** [comment]
We need to add a label indicating this issue is a goal
